thirdparty/openjdk-cacerts/zimbra-openjdk-cacerts/debian/zimbra-openjdk-cacerts.postinst

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 if [ "$1" = configure ]; then
-  /bin/chown zimbra:zimbra /opt/zimbra/common/lib/jvm/java/jre/lib/security/cacerts
-  /bin/chmod 644 /opt/zimbra/common/lib/jvm/java/jre/lib/security/cacerts
+  /bin/chown zimbra:zimbra /opt/zimbra/common/etc/java/cacerts
+  /bin/chmod 644 /opt/zimbra/common/etc/java/cacerts
```

Comment:
hmm... should we chain these two together via && so we only do step 2 (chmod) if chown is successful?